descriptors/ModuleDescriptor-template.json

```diff
@@ -683,6 +683,43 @@
             "usergroups.collection.get",
             "usergroups.item.get"
           ]
+        },
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/circulation/requests/instances",
+          "permissionsRequired": [
+            "circulation.requests.instances.item.post"
+          ],
+          "modulePermissions": [
```

Comment:
Does the system fetch the loan type for an item?

If so, we need to add those permissions in to

Reply:
no, it doesn't need loan type